presto-main/src/test/java/com/facebook/presto/type/TestMapOperators.java

```diff
@@ -778,6 +781,32 @@ public void testMapFromEntries()
     }
 
     @Test
+    public void testMapEntries()
```

Comment:
It would also be nice to add some test cases to see `map_entries` and `map_from_entries` round trip.
`assertFunction("map_from_entries(map_entries(MAP(ARRAY[1, 2, 3], ARRAY[1, 2, 3])))", .... , ....);`